docs/010 - install-sdk-go.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Install SDK"
+slug: "install-sdk-go"
+hidden: false
+createdAt: "2019-08-21T21:13:10.351Z"
```

Comment:
Are these timestamps needed? If so, @juancarlostong during the syncing process, if there are changes, we should probably look at updating the timestamps for the `updatedAt` field

Reply:
No need to change the contents here. Merely wanna make sure that we take these into account.

Reply:
@mikeng13  yeah I think the timestamp is important, I'm guess it's what powers the 'last updated' line: 
![image](https://user-images.githubusercontent.com/42477011/81122500-358cc500-8eee-11ea-87a7-25aed62dc397.png)